SU2_CFD/include/driver_structure.hpp

```diff
@@ -355,12 +356,12 @@ class CSingleZoneDriver : public CDriver {
 
```

Comment:
No, it is not. That's because the interpolator and tranfer classes for FEM, Poisson etc etc problems were never tested (to my knowledge). The driver now has an inner loop over zones but the code won't access it and throw an error if nZone > 1since the communication structure is missing.